probe/kubernetes/client.go

```diff
@@ -609,19 +599,19 @@ func (c *client) DeleteVolumeSnapshot(namespaceID, volumeSnapshotID string) erro
 }
 
 func (c *client) ScaleUp(namespaceID, id string) error {
-	return c.modifyScale(namespaceID, id, func(scale *apiextensionsv1beta1.Scale) {
+	return c.modifyScale(namespaceID, id, func(scale *autoscalingv1.Scale) {
```

Comment:
I understand that we would not be able to achieve both implementations since method signatures are changing like this. So it looks like blocker for supporting older versions.